test/unit/test_input_sender.gd

```diff
@@ -428,6 +454,13 @@ class TestSendEvent:
 		var event = InputEventKey.new()
 		sender.send_event(event)
 		assert_eq(r.gui_event, event)
+	
+	func test_sends_event_to_gui_input_signal():
```

Comment:
Change this test to 
```gdscript
test_send_event_causes_receiver_to_emit_gui_input_signal():
	var r = autofree(HasInputEvents.new())
	watch_signals(r)
	var sender = InputSender.new(r)
	var event = InputEventKey.new()
	sender.send_event(event)
	assert_signal_emitted_with_parameters(r, 'gui_input', [event])
```
This change will test the functionality more directly by checking for the signal and what was passed to it.

Reply:
I applied your advice in [f01309b](https://github.com/bitwes/Gut/pull/544/commits/f01309bea8c2476262fb8f2e8f468b5e3f5d123d).